Automattic/simplenote-ios
# SortBy Popover: Properly Centering on iOS <13

### Fix
In this PR we're fixing a glitch in which the SortBy popover wouldn't properly appear centered, at the bottom of the screen.

Ref. #507
Thank you for spotting this @bummytime !!!

### Test
0. Launch the app on an iPad device
1. Log into your account
2. Press over the Search Bar
3. Press over the **Sort By** label

- [x] Verify the popover is effectively centered at the bottom of the screen
- [x] Verify this is valid for iOS 11 and iOS 13

### Release
These changes do not require release notes.
